migrate/20250825_create_kubernetes_node_objects.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    DB[:kubernetes_node].import(
+      [:id, :vm_id, :kubernetes_cluster_id],
+      DB[:kubernetes_cluster]
+        .join(:kubernetes_clusters_cp_vms, kubernetes_cluster_id: :id)
+        .join(:vm, id: :cp_vm_id)
```

Comment:
```suggestion
      DB[:kubernetes_clusters_cp_vms]
```

You don't need the columns from the other tables.